### PR TITLE
Setup dot files

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#
+# To depend on the standard version of sale-workflow, use:
+# sale-workflow
+#
+# To explicitely give the URL of a fork, and still use the version specified in
+# .travis.yml, use:
+# sale-workflow https://github.com/OCA/sale-workflow
+#
+# To provide both the URL and a branch, use:
+# sale-workflow https://github.com/OCA/sale-workflow branchname
```
Comment (elicoidal) [inline oca_dependencies.txt:15]: @dreispt 
I would add at least project and project-reporting

Comment (elicoidal) [inline oca_dependencies.txt:15]: Other than that, :+1: 
Thanks for the maintenance!

**Comment by elicoidal** [comment]
BTW, I have another issue: I dont have the same labels as usual. Do you know where it should be fixed?

File: oca_dependencies.txt (hunk 1 above)
Comment (dreispt) [inline oca_dependencies.txt:15]: I would prefer to have the PRs introducing dependencies doing that.

**Comment by dreispt** [comment]
I don't have access to add Labels. I think there's a script to add them, but not familiar with the process.

**Comment by dreispt** [comment]
(I'll forward port this to 9.0 once merged)

File: oca_dependencies.txt (hunk 1 above)
Comment (elicoidal) [inline oca_dependencies.txt:15]: good to me

**Comment by dreispt** [comment]
Sorry for the rush, I'm merging so that PRs can take advantage of TravisCI checks.

**Pull request merged**